pkg/exposer/generic_restore.go

```diff
@@ -261,6 +262,21 @@ func (e *genericRestoreExposer) createRestorePod(ctx context.Context, ownerObjec
 
 	var gracePeriod int64 = 0
 
+	var volumeMounts []corev1.VolumeMount = nil
```

Comment:
I suggest we create a util function in `util/kube/pvc_pv.go` for this batch of code(line 265~279) as it is used by both `csi_snapshot` and `generic_restore`

Reply:
I implemented the suggestion.